schema/artwork/index.js

```diff
@@ -173,6 +173,30 @@ const ArtworkType = new GraphQLObjectType({
         description: 'Whether work can be purchased through e-commerce',
         resolve: ({ acquireable }) => acquireable,
       },
+      is_buy_nowable: {
+        type: GraphQLBoolean,
+        description: 'When in an auction, can the work buy bought before any bids are placed',
+        resolve: ({ id, acquireable }) => {
+          return gravity(`related/sales`, { size: 1, active: true, artwork: [id] })
+            .then(_.first)
+            .then(sale => {
+              if (!sale) return [false];
+
+              gravity(`sale/${sale.id}/sale_artwork/${id}`)
+                .then(saleArtwork => [sale, saleArtwork]);
```

Comment:
Yeah, I'm doing this to modify the resolved value of the parent `then`, so that the next block has access to the `sale` as well as the `saleArtwork`